### PR TITLE
Updated bluebird to version 3

### PR DESCRIPTION
**Pull request by iranreyes**
This update includes a breaking change in the way that cancel is handling

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -2,74 +2,95 @@ var animate = require('./')
...
+		st.plan(1);
+
+		var d = {value: 0};
+		var promise = animate
+												.fromTo(d, 1, { value: 0 }, { value: 100 })
+												.finally(function () {
+													st.assert(d.value < 100, 'fromTo cancelling works')
+												});
+
+		setTimeout(function(){
```
Comment (cheapsteak) [inline test.js]: indentation's a little odd

**Comment by cheapsteak** [comment]
Thanks for the PR Iran! That is quite useful

Sorry to bug you about this but could you please do a lint and match the code style to the existing base?
( Tabs instead of spaces, and no semicolons ; - )

**Comment by iranreyes** [comment]
@cheapsteak Already done the tabs for spaces, cleaned the semicolons, removed the spaces inside the curly brackets, let dummy variables name and magic numbers like originally was

**Comment by cheapsteak** [comment]
@iranreyes I think you went the wrong direction, everything is spaces now!

**Comment by cheapsteak** [comment]
also @mattdesl would the be a patch or a major version bump since it breaks compatibility with people currently using gsap-promise with bluebird v2?

**Comment by mattdesl** [comment]
Sounds like a major bump since it could break people's code. :smile:

P.S. If you guys wanted, this could conform to Jam3(ish) style and use [semistandard](https://github.com/Flet/semistandard). You can use `semistandard-format` to auto-format all the code at once, and then add it to the `package.json` so that others get the linter on `npm test`. See [here](https://github.com/mattdesl/scrape-scripts/commit/c9d7f65eb6ce7835f00a664f066df0d0fd18427f#diff-b9cfc7f2cdf78a7f4b91a753d10865a2) for reference.

**Comment by cheapsteak** [comment]
I forked the PR and made some code style changes:
https://github.com/mattdesl/gsap-promise/pull/7

Closing this one

**Comment by iranreyes** [comment]
thx :+1:

**Pull request closed without merging**